src/services/user/registration.js

```diff
@@ -8,6 +8,8 @@ const logger = require('../../logger');
 
 const { CONSENT_WITHOUT_PARENTS_MIN_AGE_YEARS } = require('../../../config/globals');
 
+const permissionsAllowedToLogin = ['student', 'employee', 'expert', 'administrator', 'teacher'];
```

Comment:
employee is no valid DB role

Reply:
https://github.com/hpi-schul-cloud/schulcloud-server/blob/develop/backup/setup/roles.json

Reply:
done

Reply:
@Parnswir can you please verified that employee is not needed?

Reply:
@CeEv Max has said that he is not sure if this role is needed. Maybe if it is unsure we can preserve it as it was in previous PR which was approved?

Reply:
@LukasSielski: I would remove employee from the list (code) and add a comment to the ticket, that employee is not a valid role.

Reply:
Then I will remove this role.